app/schemas.py

```diff
@@ -33,12 +33,24 @@ class ProductCreate(BaseModel):
 
 class ProductBase(BaseModel):
```

Comment:
I made a miss here, we could inherit from ProductCreate like the other schemas

Reply:
maybe do this in a seperate PR (inherit + pydantic description/examples)

Reply:
done here : https://github.com/openfoodfacts/open-prices/pull/58
merge 58 first then rebase ? :innocent:

Reply:
I'm not in favor of multiplying PRs at this stage of the project, as it's more time-consuming. We can do rebase when merging the PR to have clean git histories.

Reply:
Ok  I didn't understand you wanted to create it yourself ^^". I updated the PR to only keep serving images

Reply:
PRs might be slightly time consuming, but they help when multiple people are working on the project at the same time, avoiding conflitcts, and clarifying what changes were made :)